Cython/Tests/TestCodeWriter.py

```diff
@@ -123,6 +123,71 @@ def f(x, y, z):
                         return 1234
                """)
 
+    def test_assert(self):
+        self.t(u"""
+                    assert a == 10
+                    assert a == 10, 'detail message'
+               """)
+
+    def test_global_node(self):
+        self.t(u"""
+                    global a, b
+               """)
+
+    def test_nonlocal_node(self):
+        self.t(u"""
+                    nonlocal a, b
+               """)
+
+    def test_yield(self):
+        self.t(u"""
+                    yield (1, 2)
+                    yield from [1, 2, 3, 4, 5]
+               """)
+
+    def test_del(self):
+        self.t(u"""
+                    del a
+                    del a[0]
+               """)
+
+    def test_lambda(self):
+        self.t(u"""
+                    lambda x : x + 2
+               """)
+
+    def test_cascaded_cmp_node(self):
+        self.t(u"""
+                    5 < 6 <= 7 > 4 >= 3 > 2 != 1
+               """)
+
+    def test_raise(self):
+        self.t(u"""
+                    raise RuntimeError
```

Comment:
`raise ... from ...` would be nice (but I don't think it's really necessary for this to be accepted)

Reply:
Thanks for your advice,  following test cases have been added.
```python
raise RuntimeError('Something bad happened') from exc
raise RuntimeError('Something bad happened') from None
```
And still keep this test `raise RuntimeError`, because I think it meets the syntax.